src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurity.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurity
+    {
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurity_ReleaseBuffer")]
+        internal static extern Status ReleaseBuffer(
+            out Status minorStatus,
+            IntPtr bufferHandle);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurity_CopyBuffer")]
+        internal static extern Status CopyBuffer(
+            SafeGssBufferHandle handle,
+            byte[] buffer,
+            int offset);
```

Comment:
It is better to pass the capacity also and add an assert in unmanaged code to verify there is enough to copy gss_buffer_desc.length bytes